pkg/audit/manager.go

```diff
@@ -363,7 +364,12 @@ func (am *Manager) auditResources(
 
 				for _, obj := range objList.Items {
 					objNamespace := obj.GetNamespace()
-					if am.skipExcludedNamespace(objNamespace) {
+					isExcludedNamespace, err := am.skipExcludedNamespace(obj.DeepCopyObject())
```

Comment:
Deepcopying adds insurance that the object will not be modified, but increases RAM and CPU usage.

Should we assume that `skipExcludedNamespaces` will never modify the inbound object to avoid incurring this cost?

Reply:
hmm should we pass in `unstructured` to `IsNamespaceExcluded`?

Reply:
Isn't that what we're doing here currently?

unstructured implements the `runtime.Object` interface, so it is allowed.

Reply:
i meant to pass it as `unstructured.Unstructured`, and not using `DeepCopyObject` and use it as `runtime.Object`

I am okay to not modify object as part of skip

Reply:
+1 to dropping `DeepCopyObject()`

Reply:
updated, lmk what you think